README.md

```diff
@@ -210,6 +210,22 @@ The full process is described graphically in the [workflow diagram](./docs/workf
 
   > **NOTE** ⏳ This command can take few minutes or immediately depending on your internet connection or model is cached. If you have issues connecting to Hugging Face, refer to the [Hugging Face discussion forum](https://discuss.huggingface.co/) for more details.
 
+  #### Downloading a specific model from a Hugging Face repository
+
+- Specify repository, model, and a Hugging Face token if necessary
+
+  ```shell
+  HF_TOKEN=<YOUR HUGGINGFACE TOKEN GOES HERE> ilab download --repository=TheBloke/Mixtral-8x7B-Instruct-v0.1-GGUF --filename=mixtral-8x7b-instruct-v0.1.Q4_K_M.gguf
+  ```
+
+  #### Downloading an entire Hugging Face repository
+
+- Specify repository, and a Hugging Face token if necessary. For example:
+
+  ```shell
+  HF_TOKEN=<YOUR HUGGINGFACE TOKEN GOES HERE> ilab download --repository=mistralai/Mixtral-8x7B-v0.1
```

Comment:
Any reason why this example is using a different repository than the one shown above?

Reply:
This repository is for downloading the full model, and all of it's `.safetensors` files that need to be converted to serve or chat with. The above download command downloads the `.gguf` model that can be instantly served and chatted with by ilab. The Mistralai organization on HF doesn't have a copy of the .gguf file does the same way the other repo does.